Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Program.cs

```diff
@@ -30,11 +30,7 @@ static void Main(string[] args)
 using Microsoft.Extensions.DependencyInjection;
 
 var host = new HostBuilder()
-    .ConfigureFunctionsWorkerDefaults()
-    .ConfigureServices(services => {
-        services.AddApplicationInsightsTelemetryWorkerService();
-        services.ConfigureFunctionsApplicationInsights();
-    })
+    .ConfigureFunctionsWebApplication()
```

Comment:
Discussed offline: may be better to pull in unnecessary dependencies for non-http apps than to have customers trying out ASP.NET integration entering a broken state.

Reply:
We should still wire up App Insights though
```suggestion
    .ConfigureFunctionsWebApplication()
    .ConfigureServices(services => {
        services.AddApplicationInsightsTelemetryWorkerService();
        services.ConfigureFunctionsApplicationInsights();
    })
```

Reply:
wait actually isn't this a file that needs a conditional based on netcore overall? It no longer serves both types of targets. So that suggestion in my previous comment is off. But I thought this file used to be conditional to account for the debugger enablement anyway?